modules/core/src/scripting/deckgl.ts

```diff
@@ -47,6 +48,8 @@ type DeckGLProps = DeckProps & {
   container?: Element;
   /** base map library, mapboxgl or maplibregl */
   map?: any;
+  viewState?: MapViewState | null;
```

Comment:
Nice! This clears up some of the main differences between environments

Reply:
Or wait, can't you use any View in scripting? Or is it restricted to a single MapView?